pyabc2/pitch.py

```diff
@@ -498,6 +498,28 @@ def _repr_html_(self):
         cn = self.to_pitch_class()._repr_html_()
         return f"{cn}<sub>{self.octave}</sub>"
 
+    def helmholtz(self) -> str:
+        """Note name in Helmholz pitch notation, e.g. "C," and "c'"."""
+        little_c_octave = self.octave - 3
+        if little_c_octave >= 0:
+            return self.class_name.lower() + "'" * little_c_octave
+        big_c_octave = -self.octave + 2
+        return self.class_name + "," * big_c_octave
+
+    @classmethod
+    def from_helmholtz(cls, helmholtz_name: str) -> "Pitch":
+        helmholtz_name = helmholtz_name.strip()
+        is_upper = helmholtz_name[0].isupper()
+        helmoltz_re = r"([^,]+)(,*)" if is_upper else r"([^']+)('*)"
```

Comment:
It might better to be more explicit here, use `[A-G]` and `[a-g]` etc. As it is now, `Pitch.from_helmholtz("c,")` passes the matching here but errors in `.from_name` in a way that is a bit confusing, for example.

Reply:
Also might as well add a short docstring if you don't mind.